### ISSUE TITLE
Hosting Update Cache Question

### ISSUE THREAD

**Issue by MattWlodarski** [aws-amplify/amplify-console#180]
I'm hosting a simple angular website with amplify hosting. Whenever I publish an update I need to manually clear browser cookes/site data to view the update in my browser otherwise I just get the older version of the website. Whats the best way to automate this?

Thanks! #

**Comment by swaminator** [comment]
@MattWlodarski you should be able to run cmd+shift+R or kill a service worker in the browser. Amplify Console doesn't control browser updates if service workers are running.

**Comment by MattWlodarski** [comment]
Sorry let me rephrase my question. I want to be able to push an update to all the users using my app without also needing to instruct them to clear their browser cookies/side data so that they can see the newest update. In the past I've done this with firebase hosting by adding this cache control header: 
`  "hosting": {
    "public": "www",
    "headers": [
      {
        "source": "/**",
        "headers": [
          {
            "key": "Cache-Control",
            "value": "no-cache, no-store, must-revalidate"
          }
        ]
      }
    ],`

I'm not sure how to set up something similar if I have a cloudfront hosted app through amplify.

**Comment by MattWlodarski** [comment]
Ok so I found out that you need to add the --invalidateCloudFront flag to the amplify publish command to invalidate the cache. I was able to see the update on Mozilla, but Chrome still gives me the older version. If I use the cloudfront url it works on Chrome but If I use the custom domain name I created through Route53, Chrome keeps pulling up the older version.
